src/python/test/test_extract_assay.py

```diff
@@ -87,9 +88,7 @@ def test_basic_filter_allele(self):
         }
 
         self.assertEqual(
-            basic_filter(
-                table, friendly_name, values, self.proj_id, genome_reference
-            ),
+            basic_filter(table, friendly_name, values, self.proj_id, genome_reference),
```

Comment:
Please remove the linter changes here and other places if not necessary! Otherwise LGTM!